pyproject.toml

```diff
@@ -68,7 +68,8 @@ dev = [
     "setuptools==66",
     "Sphinx==6.1.3",
     "sphinx-copybutton==0.5.2",
-    "wheel==0.38.4"
+    "wheel==0.38.4",
+    "outlines==0.0.43"
```

Comment:
is this a dev dependency?

Reply:
I wasn't sure where to add it but it's used in the json grammar to regex conversion... I just wanted to make sure that pip install -e . would include outlines. Was that assumption wrong?

Reply:
i think dev dependencies are things which are only needed for developing like pytest and stuff. for the app itself it should go in the main dependency group instead. i think pip install -e . doesn't install dev dependencies but i might be wrong

Reply:
Verified that it works in main dependencies so will be updating that in next commit :)